sumOfOther.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function sumOfOther(arg) {
```

Comment:
Complexity of this function is O( n^2 ), so I suggest to sum all array items first and then create new array with subtracted items.

Reply:
@anastasiaprivalova done